deployer/src/handlers/mod.rs

```diff
@@ -554,16 +555,24 @@ pub async fn start_deployment(
     )
 )]
 pub async fn get_logs(
-    Extension(persistence): Extension<Persistence>,
+    Extension(deployment_manager): Extension<DeploymentManager>,
+    Extension(claim): Extension<Claim>,
     Path((project_name, deployment_id)): Path<(String, Uuid)>,
 ) -> Result<Json<Vec<LogItem>>> {
-    if let Some(deployment) = persistence.get_deployment(&deployment_id).await? {
+    let mut logs_request: tonic::Request<LogsRequest> = tonic::Request::new(LogsRequest {
+        deployment_id: deployment_id.to_string(),
+    });
+
+    logs_request.extensions_mut().insert(claim);
+
+    let mut client = deployment_manager.logs_fetcher().clone();
+    if let Ok(logs) = client.get_logs(logs_request).await {
+        // TODO: awaits on the From impl for `shuttle_proto::logger::LogItem` -> `shuttle_common::LogItem`.
```

Comment:
@iulianbarbu this todo is done, right?

Reply:
Yup, in theory. We should test it though, it's the main subject of this PR.

Reply:
This PR seems to update deployer correctly. So we can merge this and make another tickets if something is not working.

Reply:
I agree.